README.md

```diff
@@ -288,18 +316,29 @@ end
 after doing this you will get:
 
 - A modified `/your_api/config/routes.rb` file with the new resource:
-  ```ruby
-  Rails.application.routes.draw do
-    scope path: '/api' do
-      api_version(module: 'Api::V1', path: { value: 'v1' }, defaults: { format: 'json' }) do
-        resources :blogs
+  - Exposed mode:
+    ```ruby
+    Rails.application.routes.draw do
+      scope path: '/api' do
+        api_version(module: 'Api::V1', path: { value: 'v1' }, defaults: { format: 'json' }) do
```

Comment:
What's the difference between the `scope` used here and the `namespace` used for internal?

Reply:
`namespace` tries to match the controller modules, I think `scope` can be anything.